app/_src/mesh/features/meshopa.md

```diff
@@ -0,0 +1,668 @@
+    type: system.kuma.io/secret
+    ```
+
+1.  Pass the Secret to `OPAPolicy`:
+
+    ```yaml
+    apiVersion: kuma.io/v1alpha1
+    kind: MeshOpa
+    mesh: default
+    metadata:
+      name: opa-1
+    spec:
+      targetRef:
+        kind: Mesh
+      default:
+        appendPolicies:
+          - secret: opa-policy
+    ```
+
+{% endnavtab %}
+{% navtab Universal %}
+
+1.  Define a Secret with a policy that's Base64-encoded:
+
+    ```yaml
+    type: Secret
+    name: sample-secret
+    mesh: default
+    data: cGFja2FnZSBlbnZveS5hdXRoegoKaW1wb3J0IGlucHV0LmF0dHJpYnV0ZXMucmVxdWVzdC5odHRwIGFzIGh0dHBfcmVxdWVzdAoKZGVmYXVsdCBhbGxvdyA9IGZhbHNlCgp0b2tlbiA9IHsidmFsaWQiOiB2YWxpZCwgInBheWxvYWQiOiBwYXlsb2FkfSB7CiAgICBbXywgZW5jb2RlZF0gOj0gc3BsaXQoaHR0cF9yZXF1ZXN0LmhlYWRlcnMuYXV0aG9yaXphdGlvbiwgIiAiKQogICAgW3ZhbGlkLCBfLCBwYXlsb2FkXSA6PSBpby5qd3QuZGVjb2RlX3ZlcmlmeShlbmNvZGVkLCB7InNlY3JldCI6ICJzZWNyZXQifSkKfQoKYWxsb3cgewogICAgaXNfdG9rZW5fdmFsaWQKICAgIGFjdGlvbl9hbGxvd2VkCn0KCmlzX3Rva2VuX3ZhbGlkIHsKICB0b2tlbi52YWxpZAogIG5vdyA6PSB0aW1lLm5vd19ucygpIC8gMTAwMDAwMDAwMAogIHRva2VuLnBheWxvYWQubmJmIDw9IG5vdwogIG5vdyA8IHRva2VuLnBheWxvYWQuZXhwCn0KCmFjdGlvbl9hbGxvd2VkIHsKICBodHRwX3JlcXVlc3QubWV0aG9kID09ICJHRVQiCiAgdG9rZW4ucGF5bG9hZC5yb2xlID09ICJhZG1pbiIKfQoK
+    ```
+
+1.  Pass the Secret to `MeshOpa`:
+
+    ```yaml
+    apiVersion: kuma.io/v1alpha1
+    kind: MeshOpa
+    mesh: default
+    metadata:
+      name: opa-1
+    spec:
+      targetRef:
+        kind: Mesh
+      default:
+        appendPolicies:
+          - secret: opa-policy
+    ```
+
+{% endnavtab %}
+{% endnavtabs %}
+
+## Configuration
+
+{{site.mesh_product_name}} defines a default configuration for OPA, but you can adjust the configuration to meet your environment's requirements.
+
+The following environment variables are available:
+
+| Variable                   | Type      | What it configures     | Default value {:width=25%:}   |
+| -------------------------- | --------- | --------------------------------------| ------------------- |
+| KMESH_OPA_ADDR             | string    | Address OPA API server listens on     | `localhost:8181`    |
+| KMESH_OPA_CONFIG_PATH      | string    | Path to file of initial config        | N/A                 |
+| KMESH_OPA_DIAGNOSTIC_ADDR  | string    | Address of OPA diagnostics server     | `0.0.0.0:8282`      |
+| KMESH_OPA_ENABLED          | bool      | Whether `kuma-dp` starts embedded OPA | true                |
+| KMESH_OPA_EXT_AUTHZ_ADDR   | string    | Address of Envoy External AuthZ service | `localhost:9191`  |
+| KMESH_OPA_CONFIG_OVERRIDES | strings   | Overrides for OPA configuration, in addition to config file(*) | `[plugins.envoy_ext_authz_grpc. query=data.envoy.authz.allow]` |
+
+{% navtabs %}
+{% navtab Kubernetes %}
+
+You can customize the agent in either of the following ways:
+
+- Override variables in the data plane proxy config:
+{% navtabs %}
+{% navtab kumactl %}
```

Comment:
```suggestion
{% navtabs codeblock%}
{% navtab kumactl %}
```

Reply:
I changed this to a codeblock with tabs for kumactl and Helm so it would read a little easier.